LICENSE

```diff
@@ -1,5 +1,4 @@
-Copyright (c) IBM Corp. 2017. All Rights Reserved.
-Node module: loopback4-extension-starter
+Node module: @loopback/mqtt
```

Comment:
Got your response on the monorepo.  I guess my question was originated from this line.  
In this case, would it be `Node module: loopback4-extension-mqtt`